tools/enable_mount_volumes.py

```diff
@@ -16,7 +16,7 @@
 import logging
 import os
 import os.path
-import pprint
+#import pprint
```

Comment:
I could do. We have a bunch of commented pprint lines, which is why i commented this too (not currently used), but didn't evaluate whether those lines are useless.